# Velodyne LIDAR scan packets.

Header           header      
VelodynePacket[] packets  
